service_configuration_lib/spark_config.py

```diff
@@ -1046,6 +1061,21 @@ def get_spark_conf(
         # is str type.
         user_spark_opts = _convert_user_spark_opts_value_to_str(user_spark_opts)
 
+        if self.mandatory_default_spark_srv_conf.get('spark.yelp.jira_ticket.enabled') == 'true':
+            needs_jira_check = os.environ.get('USER', '') not in ['batch', 'TRON', '']
```

Comment:
nit: can be simplified as follows, and add a comment
```suggestion
            # Check Jira ticket if the run is not a Tron run
            needs_jira_check = os.environ.get('USER') not in ['batch', 'TRON']
```